homeassistant/components/hdmi_cec/__init__.py

```diff
@@ -374,10 +374,31 @@ class CecEntity(Entity):
     def __init__(self, device, logical) -> None:
         """Initialize the device."""
         self._device = device
-        self._icon = None
         self._state: str | None = None
         self._logical_address = logical
         self.entity_id = "%s.%d" % (DOMAIN, self._logical_address)
+        self._attr_should_poll = False
+        self._set_attr_name()
+        self._attr_icon = (
+            ICONS_BY_TYPE.get(self._device.type)
+            if self._device.type in ICONS_BY_TYPE
```

Comment:
Please avoid ternary operator expression if it spans more than one line.

Reply:
Corrected.